src/citrine/gemtables/variables.py

```diff
@@ -421,13 +422,15 @@ def __init__(self, *,
                  process_template: LinkByUID,
                  ingredient_name: str,
                  quantity_dimension: IngredientQuantityDimension,
-                 type_selector: DataObjectTypeSelector = DataObjectTypeSelector.PREFER_RUN):
+                 type_selector: DataObjectTypeSelector = DataObjectTypeSelector.PREFER_RUN,
+                 unit: str = None):
```

Comment:
```suggestion
                 unit: Optional[str] = None):
```

Reply:
Updated the docstring, and added the unit to the test case.

With regards to adding the unit to IngredientQuantityInOutput, maybe that should be a separate story, as this is focused on unblocking default table generation.